### ISSUE TITLE
Filtering, Better Sorting, PCGW/Steam URLs

### ISSUE THREAD

**Issue by elephantor** [MachineGunnur/GOG-Games#325]
Filter by Features: Single-player, Multi-player, Co-op, Achievements, Leaderboards, Controller support, In development, Cloud saves, Overlay.

Filter by Languages, Audio and/or Text: BR, CN, CZ, DE, EN, ES, FR, HU, IT, JP, KO, PL, RU, TH, TR, ZH… 

Extended Sorting by: Release Date, Total Size, Goodies or Game Size, GOG User Rating, Steam User Rating, ESRB Rating. Count all three GOG genre markings per game, not only prime one.

Add PCGamingWiki when available by:
`https://pcgamingwiki.com/api/gog.php?page=`[ID]

Add Steam and SteamDB by calling:
`https://gamesdb.gog.com/platforms/gog/external_releases/`[ID]

For example, GOG GamesDB [API](https://gamesdb.gog.com/platforms/gog/external_releases/1895572517) of "AI War 2" has:
```
                "platform_id": "steam",
                "external_id": "573410",
```
Therefore Steam and SteamDB pages are:
https://store.steampowered.com/app/573410/
https://steamdb.info/app/573410/

**Comment by Felipefpl** [comment]
Also, horror and point n click could be added too. ;)

**Comment by elephantor** [comment]
Yes, a part of the markings. Examples: [Horror](https://www.gog.com/game/darkwood), [Point-and-click](https://www.gog.com/game/brassheart).
